apis/quay/v1/quayregistry_types.go

```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 	"strings"
 
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
can we keep it as `corev1`?

Reply:
Sure, but why?

Reply:
It will make this PR smaller and easier to review.

Also, that's the way this package is used in the project:

```
$ ag k8s.io/api/core/v1 --ignore ./vendor --ignore 'zz_generated.*' --nobreak --nofilename | sort | uniq -c
  25 	corev1 "k8s.io/api/core/v1"
```

If you have a good reason to change it, open a separate PR.

Reply:
Yep, it should be `corev1` (even though I have already seen solely `v1` in some places here)